lib/init.coffee

```diff
@@ -1,8 +1,54 @@
+{CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+XRegExp = require('xregexp').XRegExp
+
 module.exports =
   config:
-    xmllintExecutablePath:
+    executablePath:
       type: 'string'
-      default: ''
-
+      title: 'xmllint Executable Path'
+      default: 'xmllint'
   activate: ->
-    console.log 'activate linter-xmllint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-xmllint.xmllintExecutablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    @executablePath = '/usr/bin/xmllint'
```

Comment:
Sorry, that was left over from debugging. I should have checked the diff more thorough before creating the PR. Removed.